addons/point_of_sale/controllers/main.py

```diff
@@ -49,14 +49,15 @@ def pos_web(self, config_id=False, **k):
 
         if not pos_session:
             return request.redirect('/web#action=point_of_sale.action_client_pos_menu')
+
+        pos_session.login()
```

Comment:
why do you do that? You should not change the way the login number is passed, because if two session are launched at the same time, they'll have the same number. See the original commit: https://github.com/odoo/odoo/commit/c0a897d528ce5882b46ed197eacb2bbddf45c4c8

Reply:
fixed